aspnetcore/grpc/why-migrate-wcf-to-dotnet-grpc.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Why migrate WCF to ASP.NET Core gRPC
+author: markrendle
+description: A summary of why ASP.NET Core gRPC is a good fit for WCF developers who want to migrate to modern architectures and platforms.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: wpickett
+ms.date: 09/02/2020
+no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: grpc/wcf
+---
+
+# Why we recommend gRPC for WCF developers
+
+This article provides a summary outlining why ASP.NET Core gRPC is a good fit for WCF developers who want to migrate to modern architectures and platforms.
```

Comment:
I think we need to mention that WCF is not being ported to .NET Core/.NET 5 and gRPC is our recommended alternative.

@shirhatti What is the best way to say this?

Reply:
Thanks for throwing the hot potato my way 😅

If it's only a brief introduction, I'm fine with the verbiage. I took a stab at making it a little stronger

> The release of .NET Core 3.0 marks a shift in the way that Microsoft delivers remote communication solutions to developers who want to deliver services across a range of platforms. .NET Core (and .NET 5) will **not** offer server-side support for the Windows Communication Foundation (WCF). With the release of ASP.NET Core 3.0, it does however provide built-in gRPC functionality.


If we want to go into more detail, this is probably the story we should tell
![image](https://user-images.githubusercontent.com/4734691/92156845-69192680-edde-11ea-8ebe-5627127e2265.png)

Reply:
@shirhatti , Perfect, thanks!  If it is OK, I will pull in just this text, but also update it in the eBook to include it (I will create a new tracking issue), where this artwork would also reside.  Since this topic was meant to be a quick summary, that points to the eBook for details.  Then we are not keeping the artwork up in several languages in more than one place.

Reply:
Just including the text for now is fine.